client/js/admin.js

```diff
@@ -43,6 +44,18 @@ var submit = function() {
 	var content = editor.getValue()
 	var tags = $("#tags").val().split(" ")
 	var title = $("#title").val()
+
+	var deleting = false
+
+	if (!(content || tags.length > 1 || tags[0] || title)) {
```

Comment:
The suggested edit above would remove the `tags[0]` clause from this.

Also I'm hesitant to use string truthiness.  I'd probably do regex whitespace matches instead.